drivers/telescope/CMakeLists.txt

```diff
@@ -194,7 +194,7 @@ install(TARGETS indi_skycommander_telescope RUNTIME DESTINATION bin)
 add_executable(indi_dsc_telescope
     dsc.cpp)
 
-target_link_libraries(indi_dsc_telescope indidriver AlignmentDriver)
+target_link_libraries(indi_dsc_telescope AlignmentDriver indidriver)
```

Comment:
What was the problem that you changed the linking order?

Reply:
DLL linking was failing with original order. Inverting the order fixed it. I thought about chaging other similar 'target_link-libraries' sentences involving indidriver but I decided not to touch what was already working